src/routers/root.js

```diff
@@ -70,7 +71,7 @@ module.exports = async (server, opts) => {
         return newTarget
       })
 
-      watchCollection.updateOne({ _id }, { $set: { targets } })
+      watchCollection.updateOne({ _id }, { $set: { targets, checkedAt } })
```

Comment:
Bỏ dòng trên đi, viết trực tiếp vào đây thôi:

```suggestion
      watchCollection.updateOne({ _id }, { $set: { targets, checkedAt: new Date() } })
```